src/Doctor/Doctor.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Dock\Doctor;
+
+class Doctor
+{
+    /**
+     * @var array
+     */
+    private $tasks;
+
+    /**
+     * @param array $tasks
+     */
+    public function __construct(array $tasks)
+    {
+        $this->tasks = $tasks;
+    }
+
+    public function examine($dryRun)
+    {
+        foreach ($this->tasks as $task) {
+            $task->run($dryRun);
```

Comment:
We should continue [there](https://github.com/inviqa/dock-cli/pull/57#issuecomment-129773351)